src/test/java/io/pinecone/PineconeIndexOperationClientTest.java

```diff
@@ -170,4 +170,56 @@ public void testCreateIndexWithAllFields() throws IOException {
         verify(mockClient, times(1)).newCall(any(Request.class));
         verify(mockCall, times(1)).execute();
     }
+
+    @Test
+    public void testDescribeIndex() throws IOException {
+        String indexName = "testIndex";
+        String indexJsonString = "{\"database\":{\"name\":\"testIndex\",\"metric\":\"cosine\",\"dimension\":3," +
```

Comment:
Did you have to create this by hand? Yikes. 😬

Reply:
Discussed internally on slack!!